cmd/rukpakctl/cmd/run.go

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"hash/fnv"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	"github.com/operator-framework/rukpak/internal/rukpakctl"
+	"github.com/operator-framework/rukpak/internal/util"
+)
+
+// newRunCmd creates the run command
+func newRunCmd() *cobra.Command {
+	var (
+		systemNamespace                      string
+		uploadServiceName                    string
+		caSecretName                         string
+		bundleDeploymentProvisionerClassName string
+		bundleProvisionerClassName           string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "run <bundleDeploymentName> <bundleDir>",
```

Comment:
WDYT about making the `bundleDeploymentName` optional, and just using the name of the bundle directory? It makes the UX even more easy

Reply:
The main reason I chose that is to ensure that the pivot workflow works regardless of whether you use the same directory name. If you had `./my-bundle-v0.1.0` and `./my-bundle-v0.2.0`, it would be really easy to create 2 different BDs when what you really want is to re-use the same BD, and pivot it from v0.1.0 to v0.2.0.